tests/unit/flask/test_databases.py

```diff
@@ -6,6 +6,7 @@
 import unittest.mock
 
 import pytest
+from ops.testing import Harness
```

Comment:
I think this isn't used anywhere in this file :D

Reply:
Yes, indeed, I will remove this, thanks!